src/sentry/features/__init__.py

```diff
@@ -179,6 +179,7 @@
 default_manager.add("organizations:session-replay-sdk-errors-only", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:session-replay-recording-scrubbing", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:session-replay-index-subquery", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:session-replay-weekly-email", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
i'm guessing we can't use flagr for this, as it's too high volume. can use `INTERNAL` and we can specify an options based handler in getsentry